opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -217,10 +228,39 @@ def export(
             # Cloud Monitoring API allows, for any combination of labels and
             # metric name, one update per WRITE_INTERVAL seconds
             updated_key = (metric_descriptor.type, record.labels)
-            last_updated_seconds = self._last_updated.get(updated_key, 0)
-            if seconds <= last_updated_seconds + WRITE_INTERVAL:
+            last_updated_time = self._last_updated.get(updated_key, 0)
+            last_updated_time_seconds = last_updated_time // 1e9
+            if seconds <= last_updated_time_seconds + WRITE_INTERVAL:
                 continue
-            self._last_updated[updated_key] = seconds
+
+            if (
```

Comment:
Now thats theres quite a bit more logic about the timeSeries Point creation, can you move that out to  seperate function?

Reply:
moved